flyteidl/protos/flyteidl/core/tasks.proto

```diff
@@ -131,6 +131,9 @@ message TaskMetadata {
     // task creates a k8s Pod. If this value is set, the specified PodTemplate will be used instead of, but applied
     // identically as, the default PodTemplate configured in FlytePropeller.
     string pod_template_name = 12;
+
+    // cache_ignore_input_vars is the input variables that should not be included when calculating hash for cache.
+    repeated string cache_ignore_input_vars  = 13;
```

Comment:
can you run `make generate` from the `flyteidl` subdirectory to produce the stubs?

Reply:
Pushed!